src/jobs/BasicGibbsJob.jl

```diff
@@ -93,16 +93,16 @@ BasicGibbsJob(
 ) =
   BasicGibbsJob(model, dpindex, dpjob, range, v0, outopts, verbose, check)
 
-function BasicGibbsJob{S<:VariableState}(
+function BasicGibbsJob(
   model::GenericModel,
-  dpjob::Dict,
-  range::BasicMCRange,
-  v0::Dict{Symbol, S};
-  dpindex::IntegerVector=find(v::Variable -> isa(v, Parameter) || isa(v, Transformation), model.vertices),
-  outopts::Dict=Dict([(k, Dict(:destination=>:nstate, :monitor=>[:value])) for k in keys(model.vertices[dpindex])]),
-  verbose::Bool=false,
-  check::Bool=false
```

Comment:
Thanks @ararslan, I did as you suggested, and then merged. Is there anyone from the Julia devs who would be willing to help me with #167 please? (cc @chris-nemeth)